src/TransactionTestsFiller/ttEIP3860/DataTestNotEnoughGasInitCodeFiller.yml

```diff
@@ -0,0 +1,20 @@
+# Creation transaction with gas limit not enough to cover initcode charge of EIP-3860
+DataTestNotEnoughGasInitCode:
+    expectException:
+        Merge: ""
+        Merge+3860: "TR_IntrinsicGas"
```

Comment:
add Merge+3860 in `additionalForks` section that is missing here. the test is not generated for this fork.